src/server/pfs/cmds/cmds.go

```diff
@@ -74,7 +75,16 @@ or type (e.g. csv, binary, images, etc).`,
 	}
 	commands = append(commands, cmdutil.CreateDocsAliases(repoDocs, "repo", " repo$", repos))
 
-	var description, project string
+	cfg, err := config.Read(false, true)
+	if err != nil {
+		return nil, err
+	}
+	_, pachCtx, err := cfg.ActiveContext(true)
+	if err != nil {
+		return nil, err
+	}
+	var description string
+	project := pachCtx.Project
```

Comment:
what if the Project isn't set in the context?

Reply:
[config.ActiveContext](https://github.com/pachyderm/pachyderm/blob/f9a8dc63e8d32a5fbe198ac97de5d02c799d5419/src/internal/config/config.go#L70-L72) (called on line 82) sets Project in the context.